DataConnectors/SquadraTechnologies.SecRMM.json

```diff
@@ -0,0 +1,119 @@
+      "legend": "Integer",
+      "baseQuery": "secRMM_CL extend count1=iff(Event_s == 'SERIAL # AUTHORIZATION', 1, 0)|summarize ERRORS=sum(count1)"
+    }
+  ],
+  "sampleQueries": [
+    {
+      "description": "Alert on USB removable storage devices ONLINE events",
+      "query": "secRMM_CL | where Event_s == 'ONLINE'"
+    },
+    {
+      "description": "Which users are writing files to removable storage devices",
+      "query": "secRMM_CL | where Event_s == 'WRITE COMPLETED' | summarize count() by User_s"
+    },
+    {
+      "description": "Microsoft BitLocker Activity for removable storage devices",
+      "query": "secRMM_CL | where DeviceDescription_s contains 'ENCRYPTED BitLocker'"
+    },
+    {
+      "description": "Microsoft Windows Defender Activity for removable storage devices",
+      "query": "secRMM_CL | where ((Event_s == 'EXTERNAL') and (Message contains 'Microsoft Defender'))"
+    },
+    {
+      "description": "Hardware Encrypted Device Activity",
+      "query": "secRMM_CL | where DeviceDescription_s contains ' ENCRYPTED Removable Media'"
+    },
+    {
+      "description": "Users who have tried to execute macros or programs from a removable storage device",
+      "query": "secRMM_CL | where ((Event_s == 'BLOCK MACROS ON DEVICE ACTIVE') or (Event_s == 'BLOCK PROGRAMS ON DEVICE ACTIVE')) | summarize count() by User_s"
+    },
+    {
+      "description": "Removable storage devices that are not encrypted (hardware or software)",
+      "query": "secRMM_CL | where ((Event_s == 'ONLINE') and (DeviceDescription_s !contains 'ENCRYPTED'))"
+    },
+    {
+      "description": "Removable storage devices that are mounted into a Virtual Machine (from physical)",
+      "query": "secRMM_CL | where ((Event_s == 'EXTERNAL') and (Message contains 'ONLINE') and (Message contains 'Virtual Machine'))"
+    },
+    {
+      "description": "Removable storage devices that are mounted into a Virtual Machine (from virtual)",
+      "query": "secRMM_CL | where ((Event_s == 'ONLINE') and (Drive_s contains '^'))"
+    },
+    {
+      "description": "Show mobile devices that are being USB mounted",
+      "query": "secRMM_CL | where ((Event_s == 'ONLINE') and (DeviceDescription_s contains 'MOBILE'))"
+    },
+    {
+      "description": "Show mobile devices that are being USB mounted but are not MDM (Microsoft Intune) enrolled",
+      "query": "secRMM_CL | where ((Event_s == 'ONLINE') and (AdditionalProgramInfo_s contains 'Mobile device is not MDM enrolled'))"
+    }
+  ],
+  "dataTypes": [
+    {
+      "name": "secRMM_CL",
+      "lastDataReceivedQuery": "secRMM_CL | summarize Time = max(TimeGenerated) | where isnotempty(Time)"
+    }
+  ],
+  "connectivityCriterias": [
+    {
+      "type": "IsConnectedQuery",
+      "value": [
+        "secRMM_CL | summarize LastLogReceived = max(TimeGenerated) | project IsConnected = LastLogReceived > ago(30d)"
```

Comment:
This query is not formed properly. Please modify as follows to include '\n':
"secRMM_CL\n            | summarize LastLogReceived = max(TimeGenerated)\n            | project IsConnected = LastLogReceived > ago(30d)"

Reply:
HI, OK, fixed.